logic/auth.js

```diff
@@ -91,6 +91,7 @@ async function changePassword(currentPassword, newPassword, jwt) {
             // wait for lnd to boot up
             if (error.response.status === constants.STATUS_CODES.BAD_GATEWAY) {
                 await sleepSeconds(1);
+                attempt--;
 
                 // user supplied incorrect credentials
             } else if (error.response.status === constants.STATUS_CODES.FORBIDDEN) {
```

Comment:
Is the assignment below even implemented as intended?

Seems like `changePasswordStatus.forbidden = true;` should be `changePasswordStatus.unauthorized = true;` because else it does continue to loop even if the credentials were incorrect.

Reply:
Wow! Great spot, @qnjctckzz8. You're right, it should `changePasswordStatus.unauthorized = true`.